modules/deleting-cluster-integration.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cli/managing-secured-clusters.adoc
+:_content-type: PROCEDURE
+[id="deleting-cluster-integration_{context}"]
+= Deleting cluster integration
+
+.Procedure
+
+* Before deleting the cluster, ensure you have the correct cluster name that you want to remove from Central:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" cluster delete --name=_<cluster_name>_
+----
++
+[IMPORTANT]
+====
+Deleting the cluster integration does not remove {product-title-short} services running in the cluster.
+You can remove them by running the `delete-sensor.sh` script from the Sensor installation bundle.
```

Comment:
Probably worthwhile to call out that this depends on the way of installation. In operator cases, deleting the secured cluster CR is enough.